packages/react-renderer-demo/src/app/src/stackblitz-templates/suir-template.js

```diff
@@ -0,0 +1,71 @@
+export const suirHtml = `
+<html>
+  <head>
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css" />
+  </head>
+  <body>
+    <div id="root"></div>
+  </body>
+</html>
+`;
+
+export const suirCode = `import React, { Component } from 'react';
+import { render } from 'react-dom';
+import FormRenderer from '@data-driven-forms/react-form-renderer';
```

Comment:
upps, its probably in all other mappers too, but we should change this to dist/cjs too.

Reply:
good thing that stackblitz is using cjs env by default